README.md

```diff
@@ -151,7 +144,7 @@ spec:
     - install
 ```
 
-When a task starts running, it starts a pod and runs each step sequentially in a separate container on the same pod. This task happens to have a single step, but tasks can have multiple steps, and, since they run within the same pod, they have access to the same volumes in order to cache files, access configmaps, secrets, etc. As mentioned previously, tasks can receive inputs (e.g. a git repository) and produce outputs (e.g. an image in a registry).
+When a task starts running, it starts a pod and runs each step sequentially in a separate container on the same pod. This task happens to have a single step, but tasks can have multiple steps, and, since they run within the same pod, they have access to the same volumes in order to cache files, access configmaps, secrets, etc. You can specify volume using workspace. It is recommended that Tasks uses at most one writeable Workspace. Workspace can be secret, pvc, config or emptyDir.
```

Comment:
You can specify the volume using...
...that Tasks use at most...
You can specify Workspaces using secret, pvc, config or emptyDir.

Reply:
I think we were talking of pod and volume in the sentence before. So specifying volume using workspace also makes sense. What do you think?